assets/scss/components/_actions-card.scss

```diff
@@ -0,0 +1,12 @@
+.actions-card {
```

Comment:
I've had to move the scss file from it original location in the main component folder, up to here.
Its the only way I could get it to "find" the scss file to compile it 🤷‍♂️ 

Also, this file started off life as the original one committed. After moving it here I ended up pretty much removing all the styles that were there previously, and then just writing styles for stuff that we need that is different to the standard GDS Summary Card (basically just overriding a couple of the standard GDS classes)